### ISSUE TITLE
missing symlink icons on CentOS 7

### ISSUE THREAD

**Issue by jdratlif** [microsoft/vscode#83320]
- VSCode Version: 1.39.2
- OS Version: CentOS 7.7.1908

Steps to Reproduce:

1. Open a folder in VS Code with symlinks
2. Symlink icon shows up as a hollow square ouline rather than the arrow I see on other platforms (e.g. debian).

Does this issue occur when all extensions are disabled?: Yes/No

Yes

I have tested in brand new CentOS 7 and Debian 10 environments. I installed only tigervnc-server, i3, git, and vs code in both. CentOS 7 has missing symlink icons, Debian 10 does not.

Perhaps there is some missing package on CentOS I could install, but I don't know what it might be. I've tried installing tango, oxygen, and gnome icon themes, but it made no difference.

Thanks.

**Comment by bpasero** [comment]
Sounds like an icon regression to me. 

@jdratlif can you attach a screenshot?

**Comment by jdratlif** [comment]
![vscode-symlink-missing-ss](https://user-images.githubusercontent.com/5897001/67583053-d36e2e80-f718-11e9-85a8-4c00d5de08a9.png)

**Comment by jdratlif** [comment]
![vscode-symlink-missing-ss](https://user-images.githubusercontent.com/5897001/67583128-f862a180-f718-11e9-9562-5f61963e5b0f.png)

**Comment by jdratlif** [comment]
And here is the same repo open in debian 10, where the icon is present.

![vscode-symlink-debian-ss](https://user-images.githubusercontent.com/5897001/67583303-53949400-f719-11e9-929d-28ea8d8cc9eb.png)

**Comment by misolori** [comment]
Looking at the source, we don't use an icon but rather it's a unicode character `⤷`:

<img width="755" alt="image" src="https://user-images.githubusercontent.com/35271042/67604534-a7f73e00-f730-11e9-96fc-758b986337ac.png">

So i'm seeing this showing up on Mac, I wonder if the default font on CentOS doesn't support this character. Unfortunately I can't find this in our source code, @joaomoreno do you have any idea?

**Comment by jdratlif** [comment]
Not sure if it's noteworthy or not, but the display is fine in CentOS 8. Unfortunately, CentOS 8 does not have all the epel packages that CentOS 7 does, so I can't quite test the same environment. I had to use Gnome rather than i3.

**Comment by jdratlif** [comment]
Well, this is apparently my fault for using minimal installs. I built i3 for CentOS 8 to replicate my environment and I didn't have the symlink arrows again. So I looked into what packages are included with 'Server with GUI' that I used before. If I install all the fonts in the 'Fonts' group, I get the symlink arrows.

I'm not sure what font is being used, but installing the font packages fixes the problem in CentOS 7 and 8.

Not sure if the font should be a package dependency or not. The glyph is probably in several font packages, so to require a specific one might not be a good idea.

Thanks.
